hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/cache/CacheManager.java

```diff
@@ -56,43 +56,23 @@
     private final Logger log = Logger.getLogger(CacheManager.class);
     private final MsgLogger msgLog = MsgLogger.LOGGER;
 
-    private static final String PUBLISH_REQUIRE_PREFIX_PROP = "hawkular-alerts.publish-require-prefix";
-    private static final String PUBLISH_REQUIRE_PREFIX_ENV = "PUBLISH_REQUIRE_PREFIX";
     private static final String DISABLE_PUBLISH_FILTERING_PROP = "hawkular-alerts.disable-publish-filtering";
     private static final String DISABLE_PUBLISH_FILTERING_ENV = "DISABLE_PUBLISH_FILTERING";
     private static final String RESET_PUBLISH_CACHE_PROP = "hawkular-alerts.reset-publish-cache";
     private static final String RESET_PUBLISH_CACHE_ENV = "RESET_PUBLISH_CACHE";
 
-    private static final Set<String> DATA_ID_PREFIXES = new HashSet<>();
-
-    static {
-        // TODO: For now just use the built-in H Metrics prefixes as a OOB convenience, but in the
-        //       future this should maybe be configurable where a list of prefixes could be supplied
-        //       via system prop.
-        DATA_ID_PREFIXES.add("hm_a_");  // Metrics AVAILABILITY
```

Comment:
The prefixes are now known only on the metrics side of the code, because it needs to generate the proper prefixes for the dataIds.  This makes sense to me because it really is the metrics client's responsibility.  The prefixes are a convention established due to the way Hm generates its primary keys.  It would be useful documentation for Hm where we discuss how to integrate with Ha.